HiP-DataStore/Controllers/ExhibitsController.cs

```diff
@@ -230,6 +235,28 @@ public IActionResult GetReferenceInfo(int id)
             return ReferenceInfoHelper.GetReferenceInfo(ResourceTypes.Exhibit, id, _entityIndex, _referencesIndex);
         }
 
+        /// <summary>
+        /// Gets the exhibits where the current user is the owner
+        /// </summary>
+        [HttpGet("My")]
+        [ProducesResponseType(typeof(AllItemsResult<ExhibitResult>), 200)]
+        [ProducesResponseType(400)]
+        public IActionResult GetMyExhibits(ExhibitQueryArgs args)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+            try
```

Comment:
I think, that we need checking if the user is allowed to get deleted content. Because only user with special roles can.